easybuild/easyconfigs/u/unixODBC/unixODBC-2.3.12-foss-2023a.eb

```diff
@@ -0,0 +1,27 @@
+# Easyconfig for unixODBC
+# Author: Lykle Voort
+# SURFsara, Amsterdam, The Netherlands
+# Updated: Petr Král (INUITS)
+# Updated: Stefan Wolfsheimer (SURF)
+
+easyblock = 'ConfigureMake'
+
+name = 'unixODBC'
+version = '2.3.12'
+
+homepage = "https://www.unixodbc.org"
+description = """unixODBC provides a uniform interface between
+application and database driver"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['https://www.unixodbc.org/']
+checksums = ['f210501445ce21bf607ba51ef8c125e10e22dffdffec377646462df5f01915ec']
+
+sanity_check_paths = {
+    'files': ['lib/libodbc%s.so' % x for x in ['', 'cr', 'inst']],
+    'dirs': []
+}
+
```

Comment:
Could you also add one or more sanity check command(s) for some of the binaries? E.g. `odbc_config --version`?